_posts/2016-04-08-sprint3_update.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Technical Update (4/8/2016) and Spring Progress Report
+---
+
+The Hydra-in-a-Box technical team's sprint 3 demo -- showing deployment to AWS, multi-tenancy, the configuration UI, and Sufia's new batch upload workflow -- is now available:
+
+<div class="videoWrapper"> <iframe width="1280" height="720" src="https://bluejeans.com/s/9qBr/" frameborder="0" allowfullscreen></iframe></div>
```

Comment:
Done, though I'm torn re: doing this in the future because the YouTube version (from downloaded BlueJeans mp4) overlays the speaker's name atop prime real estate in the upper left-hand corner. OTOH, consistency from week to week is good, and it allows us to collect analytics in the same way every week. I'm on the fence for future weeks. Maybe a topic for an upcoming managers meeting.